docs/data-access/daco/renew-and-close.md

```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
 
 ### How to Complete an Annual Attestation
 
-1. Upon receiving the email reminder to complete the annual attestation, you can login and navigate to the correct application.
-1. At the top of the application, you will see a yellow banner with some terms to review. Please carefully review the terms and if you agree, click the “I attest to the above terms” button.
-1. At this time, the application will move back into the approved state, and if access was paused, it will resume for the project team.
+1. When you receive the email reminder for the annual attestation, log in to the DACO platform and navigate to your application.
+2. At the top of the application page, a yellow banner will be displayed, containing terms that require your review. Please read these terms carefully.
+3. After reviewing, if you agree with the terms, click the “I attest to the above terms” button.
+
+Following your attestation, the application status will revert to the approved state. If access was previously paused, it will be reinstated for your project team.
 
 ## Renewing an Application
 
-The DACO approval expires after 2 years of access. An email reminder will be sent to the applicant 90 days and 45 days prior to the access expiry date as well as on the date of expiry. If the project team is to remain active, the applicant may wish to renew beyond the 2 years. The option to renew the application will be available 90 days before the application expires, and up to 90 days after it expires.
+The DACO approval expires after 2 years of access. An email reminder will be sent to the applicant 90 days and 45 days prior to the access expiry date, as well as on the date of expiry. If the project team is to remain active, the applicant may wish to renew beyond the 2 years. The option to renew the application will be available 90 days before the application expires, and up to 90 days after it expires.
 
 ### How to Renew an Application
 
 1. When the applicant receives the email reminder that the application is about to expire (or has expired), they can login and click the **Renew** button.
+
    ![DACO renew button for an expired application](/assets/data-access/daco-renew-button.png)
-1. At this time, the application will be unlocked and you can make any changes (if necessary) to the currently approved application, agree to the Data Access Agreement and Appendices again, and re-sign and submit the renewal application.
+
+2. Upon clicking the **Renew** button, a new, separate draft renewal application is created. This draft is linked to the original application for reference purposes. The applicant must complete this new draft application for renewal. The original application cannot be edited directly.
```

Comment:
similar to comment above - maybe the note about editing the original application can specify that this is only available on still approved applications? or that edits to the original will not transfer to the renewal